src/main/io/displayport_msp_bf_compat.c

```diff
@@ -328,28 +328,28 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_ZERO_HALF_LEADING_DOT:
         return BF_SYM_ZERO_HALF_LEADING_DOT;
+*/
 
     case SYM_AUTO_THR0:
-        return BF_SYM_AUTO_THR0;
+        return 'A';
 
     case SYM_AUTO_THR1:
-        return BF_SYM_AUTO_THR1;
+        return 'T';
```

Comment:
I think it would be better to keep the throttle symbol. 

So manual throttle is $`
and auto throttle is `A$`

Reply:
Yes, you have a point there. Just pushed changes keeping the throttle symbol.